how-to/password-protect-project.rst

```diff
@@ -29,7 +29,7 @@ See :ref:`basic-auth` for more.
 How this works
 --------------
 
-These variables are handled in the :ref:`divio-cloud-sso` module.
+These variables are handled in the :ref:`Divio Cloud single-sign-on <divio-cloud-sso>` module.
```

Comment:
`Divio single-sign-on` (we don't use *Divio Cloud* any more)

Reply:
Done